Alloy/builtins/moment/lang/af.js

```diff
@@ -1,72 +1,71 @@
 //! moment.js locale configuration
+//! locale : Afrikaans [af]
+//! author : Werner Mollentze : https://github.com/wernerm
 
-;(function (global, factory) {
-   typeof exports === 'object' && typeof module !== 'undefined'
-       && typeof require === 'function' ? factory(require('/alloy/moment')) :
-   typeof define === 'function' && define.amd ? define(['../moment'], factory) :
-   factory(global.moment)
-}(this, (function (moment) { 'use strict';
+import moment from '../moment';
```

Comment:
I'm not sure if it's worthwhile being concerned about the switch to ESM here, Android should work fine and I guess babel should transpile for iOS but I'd be curious to see how this might impact build times if we transpile

Reply:
Looks like the old version used the files from `locale` instead of `dist/local`. I've change that.

And in case we'll update them later again: there is one change inside the files:
`require('../moment')` is `require('/alloy/moment')`. Can be changed with search & replace.